packages/pods-core/src/builtin/MarkdownPod.ts

```diff
@@ -353,7 +353,7 @@ export class MarkdownImportPod extends ImportPod<MarkdownImportPodConfig> {
   }
 
   static cleanAssetPath(path: string): string {
-    return path.replace(/[\\|/.]/g, "");
+    return path.toLowerCase().replace(/[\\|/.]/g, "");
```

Comment:
Won't this cause problems on case sensitive OSs like Linux/Mac? I think we should preserve the case of the file paths